applications/luci-app-smartdns/htdocs/luci-static/resources/view/smartdns/smartdns.js

```diff
@@ -0,0 +1,476 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * smartdns is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+'use strict';
+'require fs';
+'require uci';
+'require form';
+'require rpc';
+
+var conf = 'smartdns';
+var callServiceList = rpc.declare({
+	object: 'service',
+	method: 'list',
+	params: ['name'],
+	expect: { '': {} }
+});
+
+function getPidOfSmartdns() {
+	return L.resolveDefault(callServiceList(conf), {})
+		.then(function (res) {
+			var isrunning = false;
+			try {
+				isrunning = res[conf]['instances']['smartdns']['running'];
+			} catch (e) { }
+			return isrunning;
+		});
+}
+
+function getIPTablesRedirect() {
+	return fs.exec('/usr/sbin/iptables', ['-t', 'nat', '-nL', 'PREROUTING']).then(function (res) {
+		if (res.code === 0) {
+			return res.stdout.trim();
+		} else {
+			return "";
+		}
+	});
+}
+
+function getIP6TablesRedirect() {
+	return fs.exec('/usr/sbin/ip6tables', ['-t', 'nat', '-nL', 'PREROUTING']).then(function (res) {
+		if (res.code === 0) {
+			return res.stdout.trim();
+		} else {
+			return "";
+		}
+	});
+}
+
+function smartdnsServiceStatus() {
+	return Promise.all([
+		getPidOfSmartdns(),
+		getIPTablesRedirect(),
+		getIP6TablesRedirect()
+	]);
+}
+
+function smartdnsRenderStatus(res) {
+	var renderHTML = "";
+	var isRunning = res[0];
+	var ipt = res[1];
+	var ip6t = res[2];
+
+	var serverPort = uci.get_first('smartdns', 'smartdns', 'port');
+	var redirectMode = uci.get_first('smartdns', 'smartdns', 'redirect');
+	var ipv6Enabled = uci.get_first('smartdns', 'smartdns', 'ipv6_server');
+
+	if (isRunning) {
+		renderHTML += "<b><font color=green>SmartDNS - " + _("RUNNING") + "</font></b></em>";
+	} else {
+		renderHTML += "<b><font color=red>SmartDNS - " + _("NOT RUNNING") + "</font></b></em>";
+	}
+
+	if (redirectMode === "dnsmasq-upstream") {
+		var matchLine = "127.0.0.1#" + serverPort;
+		var dnsmasqServer = uci.get_first('dhcp', 'dnsmasq', 'server') || "";
+
+		if (dnsmasqServer.indexOf(matchLine) < 0) {
+			renderHTML += "<br></br><b><font color=red>" + _("Dnsmasq Forwared To Smartdns Failure") + "</font></b>";
+		}
+	} else if (redirectMode === "redirect") {
+		var redirectRules = (ipt || '').split(/\n/).filter(function (rule) {
+			return rule.match(/REDIRECT/) && rule.match(/dpt:53/) && rule.match("ports " + serverPort);
+		});
+
+		if (redirectRules.length <= 0) {
+			renderHTML += "<br></br><b><font color=red>" + _("IPV4 53 Port Redirect Failure") + "</font></b>";
```

Comment:
Please change `<br></br>` to `<br />` and `<font color=red>` to `<span style="color:red">` as well as `</font>` to `</span>`